package.json

```diff
@@ -2,6 +2,10 @@
   "name": "loopback-sdk-ios-codegen",
   "version": "0.0.0",
   "description": "Tool for auto-generating iOS client source code for LoopBack App",
+  "os": ["darwin"],
+  "engines": {
+    "node": ">=4"
```

Comment:
TBH, I think it doesn't really matter. Support for Node v0.12 will end in about a week, and since this repository hasn't reached `v1.0.0` yet, I doubt anybody will care about dropped support for pre-4 versions of Node.